harness/determined/cli/trial.py

```diff
@@ -121,6 +126,77 @@ def trial_logs(args: Namespace) -> None:
     )
 
 
+@authentication.required
+def generate_support_bundle(args: Namespace):
+    temp_dir = tempfile.TemporaryDirectory()
```

Comment:
I think we need to clean up this temp directory? maybe use a context manager or try/finally.

Reply:
> I think we need to clean up this temp directory? maybe use a context manager or try/finally.

I tested this and it does appear that the temp_dir gets cleaned up, but I agree that a context manager would make this more explicit.

Reply:
added a with statement!